src/HDInsight/HDInsight/ManagementCommands/RevokeAzureHDInsightHttpServicesAccessCommand.cs

```diff
@@ -15,10 +15,13 @@
 using Microsoft.Azure.Commands.HDInsight.Commands;
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.HDInsight.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+using System;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
+    [CmdletDeprecation(ReplacementCmdletName = "Set-AzHDInsightGatewayCredential")]
```

Comment:
same comment, is this a deprecation, or is this cmdlet currently not working?

Reply:
> same comment, is this a deprecation, or is this cmdlet currently not working?

After discussion, we have removed the two cmdlets rather than deprecated.